plugin.rb

```diff
@@ -274,6 +274,17 @@ def execute(args)
     end
   end
 
+  DiscourseEvent.on(:post_edited) do |post, topic_changed|
+    if topic_changed && SiteSetting.voting_enabled
```

Comment:
before queuing the job can you do quick check on the topic to see if it has more than 0 votes? 

also I guess this leave a whole around bulk actions, does that fire a post edited? (select multiple topics from web and then change category)

Reply:
Ok, I've made it so that it doesn't enqueue a job unless there are votes to release or reclaim.

And yes this event doesn't trigger when moving topics in bulk because we don't use PostRevisor there. Should I simply trigger the event or make bulk actions user PostRevisor?